notebooks/imdb_sentiment_analysis.livemd

```diff
@@ -0,0 +1,122 @@
+# IMDB Sentiment Analysis
+
+## Dependencies
+
+```elixir
+Mix.install([
+  {:axon, "~> 0.1.0-dev", github: "elixir-nx/axon"},
+  {:exla, "~> 0.1.0-dev", github: "elixir-nx/exla", sparse: "exla"},
+  {:nx, "~> 0.1.0-dev", github: "elixir-nx/nx", sparse: "nx", override: true},
+  {:scidata, "~> 0.1.2"}
+])
+```
+
+## Data
+
+```elixir
+%{review: reviews, sentiment: sentiment} = Scidata.IMDBReviews.download()
+
+max_seq_len = 64
+
+pad_or_slice_to_length = fn x ->
+  # This will be slow in a JIT :(
```

Comment:
Why this will be slow in a JIT? It seems this could be writable as defn?

Reply:
We are trying to normalize the data to a single shape, so its shape will change every time